.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
Actually I have the same question as in the `.gitattributes` here. What does this bring? I don't see any reason for it, especially here in this repository where there's only a Dockerfile + service code around it. No `mdx` files. Things like indent make no sense here etc. What would change and what would be better if we merge this file?

Reply:
The `.editorconfig` and `.gitattributes` are boilerplate and can definitely be reduced; however, the reason why I introuced them is because the default behavior on windows is to checkout files as crlf instead of lf. In the past this has caused problems for me.